updated readme. (#1)
dcos/prometheus-alert-rules: README.md

# DC/OS Prometheus Alert Rules

> This repository is the home of the official DC/OS Prometheus, a sub-component of the DC/OS Monitoring package, alert rules.

Reviewer: There is no `DC/OS Prometheus` component. This is a repository for the Prometheus Alert Rules for DC/OS. 

I would prefer that we explain the relationship with DC/OS Monitoring separately. This isn't really a sub-component. How about:

```suggestion
This repository is the home of the official DC/OS Prometheus alert rules. They are automatically included in releases of the DC/OS Monitoring framework. 
```

## Severity Levels

Alerts should be configured at two severity levels, warning and critical:

> - **Warning** should be used for alerts which will be read by a human, but are not

Reviewer: 'are not requiring immediate action' reads a little clumsily

```suggestion
- **Warning** should be used for alerts which will be read by a human, but do not
```

> requiring immediate action. They would, for example, trigger an email notification. 

Reviewer: ```suggestion
require immediate action. They would, for example, trigger an email notification. 
```
